src/Database/Attach/Resizer.php

```diff
@@ -485,11 +488,11 @@ protected function openImage($file)
                 break;
             case 'image/gif':
                 $img = @imagecreatefromgif($filePath);
-                self::retainImageTransparency($img);
+                self::retainImageTransparency($img, $this->extension);
                 break;
             case 'image/png':
                 $img = @imagecreatefrompng($filePath);
-                self::retainImageTransparency($img);
+                self::retainImageTransparency($img, $this->extension);
                 break;
             case 'image/webp':
                 $img = @imagecreatefromwebp($filePath); break;
```

Comment:
@mjauvin Do WEBP files also need to retain transparency?

Reply:
done